.gitignore

```diff
@@ -49,6 +50,7 @@ koreader-debian-x86_64-linux-gnu*
 koreader-debian-armel-arm-linux-gnueabi*
 koreader-debian-armhf-arm-linux-gnueabihf*
 koreader-kindle-legacy-arm-kindle-linux-gnueabi*
+koreader-kindle-arm-kindle5-linux-gnueabi*
```

Comment:
There's also kindlepw2. I'm not sure if there's any particular reason we're not using something like `koreader-android*` though. Is there a deeper reason or has it just grown this way organically because there were only a two or three targets in the beginning?

Pinging @NiLuJe @houqp

Reply:
Thanks for bringing that up. I sorta opened this hoping to start that conversation in the first place :P